tests/functional/u/undefined/undefined_variable_py38.py

```diff
@@ -123,5 +123,19 @@ def type_annotation_used_after_comprehension():
 
 def type_annotation_unused_after_comprehension():
     """https://github.com/PyCQA/pylint/issues/5326"""
+    my_int: int  # [unused-variable]
```

Comment:
```suggestion
    # TODO follow up in https://github.com/PyCQA/pylint/issues/5713
    my_int: int  # [unused-variable]
```

Reply:
I'll just go ahead and open the PR for that; it's not 100% slam-dunk that it's worth changing the message, but if there's a time to do it, it's probably soon before more existing disables out there have to be changed.